### PR TITLE
Missing bundle install in testing pages locally

### PR DESCRIPTION
**Pull request by marie-bnl**
### Why:

Without doing `bundle install` it gives
```
> bundle exec jekyll serve .
bundler: command not found: jekyll
Install missing gem executables with `bundle install`
```

### What's being changed:

image

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — APPROVED** [review]
- [ ] @main

**Review by luismedicina23 — APPROVED** [review]
- - [ main ] @luismedicina23

**Comment by janiceilene** [comment]
@victorbnl Thanks so much for opening a PR! I'll get this triaged for review 🌟

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — APPROVED** [review]

File: content/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Before you can use Jekyll to test a site, you must:
 
 {% data reusables.command_line.open_the_multi_os_terminal %}
 {% data reusables.pages.navigate-publishing-source %}
+4. Execute `bundle install`
```
Comment (shati-patel) [inline content/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll.md:33]:
```suggestion
3. Run `bundle install`.
```

**Review by shati-patel — APPROVED** [review]
Thank you for your contribution, @victorbnl, much appreciated! I'll get this merged shortly 😃

**Comment by NhatMode** [comment]
Ok dong y

**Pull request merged**